first-analysis-steps/README.md

```diff
@@ -14,5 +14,5 @@ with programming in Python.
 
 The [Software Carpentry
 workshop](http://twitwi.github.io/2015-06-02-cern-lhcb/) gives a great
-introduction to these, as well as to many other useful computing tools.
+introduction to these, as well as to many other useful computing tools. The [StarterKit](https://lhcb.github.io/analysis-essentials/index.html) has some supplementary material as well.
```

Comment:
We try to use the "Starterkit" capitalisation everywhere, rather than "StarterKit".